src/components/Person.js

```diff
@@ -60,7 +60,7 @@ export default function Person({ person, currentTag }) {
         {person.twitter && (
           <TwitterHandle>
             <a
-              href={`https://twitter.com/${person.twitter}`}
+              href={`https://twitter.com/${person.twitter.slice(1)}`}
```

Comment:
Please make a check first if twitter starts with `@` something like, otherwise it will strip first letter on usernames that were added without `@`
```js
person.twitter.startsWith('@') ? person.twitter.slice(1) : person.twitter
```

Reply:
Or even
`.replace('@', '')`

Reply:
With replace will replace even in the middle of the name `some@handle` -> `somehandle`

Reply:
Twitter handles can't have `@` in them
As per Twitter Help Center:
> A username can only contain alphanumeric characters (letters A-Z, numbers 0-9) with the exception of underscores, as noted above. Check to make sure your desired username doesn't contain any symbols, dashes, or spaces.

Reply:
Ok, agree, `.replace('@', '')` can be used then

Reply:
I agree, let's use replace. Also `replace` is used to show the actual twitter username in line [68](https://github.com/wesbos/awesome-uses/pull/400/files#diff-8cb77f435c7ded7519d20549e532dc6cL68)